### PR TITLE
doc: update information about compression

### PR DESCRIPTION
**Pull request by idobes**
Updated README: added information about supported compression types

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,22 @@ TOPICS_TO_SERVER = [
 The `name` field specifies the local name of the topic i.e., the name under which the robot provides
 certain data. The `type` field contains the textual representation of the topic's type, e.g., sensor_msgs/Image.
 
+Additionally, a `compression` field can be addded. For image topics, available compression methods are `jpeg`, `h264` and `hevc`. In case of point clouds, `draco` compression is supported. Moreover, `lz4` compression is available for other topic types. Please note that whenever a compression is requested for a particular topic, the `compression` field with the same compression type *must* be specified for both the clinet and the server (i.e. also in server's `TOPICS_FROM_CLIENT` variable).
```
Comment (ZdenekM) [inline README.md]: Typo (clinet).

File: README.md [hunk 2]
```diff
@@ -48,6 +48,22 @@ TOPICS_TO_SERVER = [
...
+TOPICS_TO_SERVER = [
+  {
+    "name": "/axis/image_raw",
+    "type": "sensor_msgs/Image",
+    "compression": "h264"
+  }
+]
+```
+
+Other client's varibales for sending services, actions, and transforms, are specified in the following way:
```
Comment (ZdenekM) [inline README.md]: Typo (varibales).

**Review by ZdenekM — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (idobes) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (idobes) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -48,6 +48,22 @@ TOPICS_TO_SERVER = [
 The `name` field specifies the local name of the topic i.e., the name under which the robot provides
 certain data. The `type` field contains the textual representation of the topic's type, e.g., sensor_msgs/Image.
 
+Additionally, a `compression` field can be addded. For image topics, available compression methods are `jpeg`, `h264` and `hevc`. In case of point clouds, `draco` compression is supported. Moreover, `lz4` compression is available for other topic types. Please note that whenever a compression is requested for a particular topic, the `compression` field with the same compression type *must* be specified for both the client and the server (i.e. also in server's `TOPICS_FROM_CLIENT` variable).
```
Comment (ZdenekM) [inline README.md]: Sorry, there is one more (addded).

**Review by ZdenekM — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (idobes) [reply README.md]: Fixed. I'll have to turn on the spell checker :D

**Review by ZdenekM — APPROVED** [review]

**Pull request merged**